clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2090,6 +2090,53 @@ be applied multiple times to the same loop.
   }];
 }
 
+def SYCLIntelFPGALoopCoalesceAttrDocs : Documentation {
+  let Category = DocCatVariable;
+  let Heading = "loop_coalesce";
+  let Content = [{
+This attribute applies to a loop. Indicates that the loop nest should be
```

Comment:
nit: 'should' most of the time indicates a design goal of the feature in general. I'm not 100% sure, that we are taking in account the goals on clang level.

Reply:
I believe 'should' is appropriate here